### DISCUSSION TITLE
markdown to orgmode:  title part of a link wrongly added a line break

### DISCUSSION THREAD

**Discussion by lywon** [jgm/pandoc#10042] [Q&A]
## original markdown format input content:
```
1. [xx xxxxxxx xxxxxxxx xx on our way home from - XXXXXXX.COM](https://www.xxxxxxx.com/video.kuuobaoc652/xx_xxxxxxx_xxxxxxxx_xx_on_our_way_home_from#)
```
## converted orgmode format output content:
```
1. [[https://www.xxxxxxx.com/video.kuuobaoc652/xx_xxxxxxx_xxxxxxxx_xx_on_our_way_home_from#][xx
   xxxxxxx xxxxxxxx xx on our way home from - XXXXXXX.COM]]
```
## expected orgmode format output content:
```
1. [[https://www.xxxxxxx.com/video.kuuobaoc652/xx_xxxxxxx_xxxxxxxx_xx_on_our_way_home_from#][xx xxxxxxx xxxxxxxx xx on our way home from - XXXXXXX.COM]]
```

## pandoc info
pandoc 3.3
Features: +server +lua
Scripting engine: Lua 5.4
User data directory: C:\Users\Administrator\AppData\Roaming\pandoc
Copyright (C) 2006-2024 John MacFarlane. Web: https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.

**Comment by jgm** [comment]
See the documentation for `--wrap`.
Pandoc will wrap output by default to fit within a certain line length.  (Semantically this should make no difference in org.)  You can avoid this with `--wrap=none` or `--wrap=preserve`.
